src/client/pages/EventPage/components/AppsOverTimeStatistics.tsx

```diff
@@ -8,7 +8,7 @@ export default class AppsOverTimeStatistics extends EventStatisticsComponent {
     const { statistics } = this.props;
 
     // Create the data array needed to make the line chart
-    const appsTimeData: { date: string; appCount: number }[] = [];
+    const appsTimeData: Array<{ date: string; appCount: number }> = [];
```

Comment:
What difference does this make? I only googled how to define array types in TS but really haven't looked into the nitty-gritty yet.

Reply:
Just for linting